examples/fabric-bridge-app/linux/ActionsServerCallbacks.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app/AttributeAccessInterface.h>
+#include <app/AttributeAccessInterfaceRegistry.h>
+#include <app/util/attribute-storage.h>
+#include <lib/support/CodeUtils.h>
+#include <lib/support/logging/CHIPLogging.h>
+
+#include <vector>
+
+#include "Device.h"
+#include "main.h"
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::Actions::Attributes;
+
+namespace {
+
+class ActionsAttrAccess : public AttributeAccessInterface
```

Comment:
Do you actually want Actions on the fabric bridge?  What actions would you support, exactly?

Reply:
A Bridge typically has information on topology or logical grouping of the Bridged Devices. If a Bridge has such information on topology or logical grouping, it SHOULD expose such information in the EndpointLists attribute of an ActionsClusterluster.

We can ignore the ActionsCluster for now, until we use it.